src/sites.rs

```diff
@@ -399,11 +399,20 @@ pub(crate) async fn sites_task(
                     log::info!("Getting sites info");
 
                     let sites = RpcCall::new("sites", "getSites")
-                        .exec(&client_cmd_tx)
+                        .exec::<_, BTreeMap::<_,_>, _>(&client_cmd_tx)
                         .await;
 
                     let (sites_info, sub_hps) = match sites {
                         Ok(sites) => {
+                            if sites
+                                .get("_meta")
+                                .map(RpcValue::as_map)
+                                .and_then(|map| map.get("type"))
+                                .map(RpcValue::as_str)
+                                .is_some_and(|type_str| type_str == "HP3")
+                            {
```

Comment:
I believe this code is wrong. Looking at our sites `_meta` in the root, the structure is:
```
  "_meta":{
    "HP3":{"type":"HP3"},
```

Therefore:
```suggestion                            if sites
                                .get("_meta")
                                .map(RpcValue::as_map)
                                .and_then(|map| map.get("HP3"))
                                .map(RpcValue::as_map)
                                .and_then(|map| map.get("type"))
                                .map(RpcValue::as_str)
                                .is_none_or(|type_str| type_str != "HP3")
                            {
```